app/models/case/base.rb

```diff
@@ -646,6 +646,10 @@ def overturned_ico?;      false;  end
   def overturned_ico_sar?;  false;  end
   def overturned_ico_foi?;  false;  end
 
+  def identifier
+    name.sub(/\sname\s?\d{0,3}$/, '')
+  end
+
```

Comment:
Shouldn't there be tests for this and what is attribute used for? Is it only for factories and testing?

Reply:
This attribute is only used for testing, but invaluable when trying to decide what cases should be in a collection and which should not.